src/components/ec-donut/ec-donut.story.js

```diff
@@ -0,0 +1,41 @@
+import { storiesOf } from '@storybook/vue';
+import { number, text } from '@storybook/addon-knobs';
+import EcDonut from './ec-donut.vue';
+
+const stories = storiesOf('Donut chart', module);
+
+stories.add('basic', () => ({
+  components: { EcDonut },
+  filters: {
+    currencyFormat(value, currency) {
+      return new Intl.NumberFormat('gb-GB', { style: 'currency', currency, currencyDisplay: 'code' }).format(value);
+    },
+  },
+  props: {
+    used: {
+      default: number('Used', 2500),
+    },
+    amount: {
+      default: number('Amount', 10000),
+    },
+    currency: {
+      default: text('Currency', 'GBP'),
```

Comment:
Because the currencyFormat filter expects valid currency codes, I'd changed this one to `select(['GBP', 'EUR', 'USD', 'CAD'], 'GBP')`